label_maker/main.py

```diff
@@ -20,9 +23,14 @@
 logger = logging.getLogger(__name__)
 
 def get_bounds(feature_collection):
-    """Get a bounding box for a FeatureCollection of Polygon Features"""
-    features = [f for f in feature_collection['features'] if f['geometry']['type'] in ['Polygon']]
-    return MultiPolygon(list(map(lambda x: Polygon(x['geometry']['coordinates'][0]), features))).bounds
+    """Get a bounding box for a FeatureCollection"""
+    features = [f for f in feature_collection['features']]
+    shape_lst = []
+    for x in np.arange(0, len(features)):
+        shape_lst.append(shape(features[x]['geometry']))
```

Comment:
@martham93 can you flatten this into one list comprehension `shapes = [shape(geom) for ...]`

Reply:
resolved thank you